CMakeLists.txt

```diff
@@ -149,7 +149,7 @@ endif()
 # they can be properly gc'd. -s: strip symbol. -fno-exceptions -fno-rtti:
```

Comment:
Looks like this could be edited as part of the same PR

Reply:
done